entity-framework/core/querying/related-data.md

```diff
@@ -109,6 +109,27 @@ Contents of `School` navigation of all People who are Students can be eagerly lo
   context.People.Include("School").ToList()
   ```
 
+### Filtered include
+
+> [!NOTE]
+> This feature was introduced in EF Core 5.0.
```

Comment:
Not sure what we do for upcoming releases, but maybe "will be introduced"? Mention the specific preview version as well?

Reply:
Will add preview version once it actually ships ;)

Reply:
I think it is fine to write introduced in 5.0. Just additional maintenance cost otherwise. The feature will also end up on https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-5.0/whatsnew to say which exact preview it was released in.